cmd/svcat/binding/get_cmd.go

```diff
@@ -45,7 +50,7 @@ func NewGetCmd(cxt *command.Context) *cobra.Command {
 	}
 
 	command.AddNamespaceFlags(cmd.Flags(), true)
-
+	command.AddOutputFlags(cmd.Flags())
```

Comment:
Because the --output flag is used by kubectl as well, you will need to add an explicit mapping for when svcat is used in plugin mode.

See https://github.com/kubernetes-incubator/service-catalog/blob/2e1ca7f8cf2b5d45b0542602f0d12cb18bb05f98/cmd/svcat/plugin/plugin.go#L60-L66 for an example of how we do that with the -v and --namespace flags.

Reply:
Does that need to happen for "--output" and "-o"? Or is it smart enough to handle that?